backend/inspirehep/editor/views.py

```diff
@@ -308,3 +314,13 @@ def authorlist_text():
         return jsonify(parsed_authors)
     except Exception as err:
         return jsonify(status=400, message=" / ".join(err.args)), 400
+
+
+@blueprint.route("/<endpoint>/<pid_value>/<version_id>/lock/resolve", methods=["GET"])
```

Comment:
```suggestion
@blueprint.route("/<endpoint>/<pid_value>/<version_id>/lock/release", methods=["POST"])
```
Not idempotent, so can't be a `GET`, and the standard terminology is `acquire`/`release`.

Or maybe even remove the version from the endpoint, and pass the whole `Etag` as a body, that can be treated as an opaque version identifier by the editor frontend? that would also allow you to write a backend test asserting that if you pass the record `Etag` to the endpoint, the lock gets cleared successfully, and catch the case where the `Etag` changes.

Reply:
Sorry, I don't understand why it is incorrect now, I am taking record version directly from etag in frontend, so what's the difference if I'll pass something like `'W 25'` or 25?

Reply:
That's not incorrect (`GET` method is). Currently the editor frontend needs to be aware of how the Etag is constructed, but I believe that's better left as a backend implementation detail.

Reply:
`etag` was not part of the refinement, and we discussed it all together. Implementing `etag` would need to change the logic and it's like starting from scratch again. I will strongly suggest to implement this feature as we designed and later to improve it if needed.

Reply:
@drjova The version was part of the refinement. The only way for the editor to get the version currently is through the Etag. So we can't leave it alone (unless we add some different mechanism to transmit the version to the editor). The question is: what should extract the version from the ETag: the record editor frontend or the backend?

Reply:
Just to add that if we want to post by using etag we'll have to encode and then decode etag in backend, what imo is less clear than doing a request with a version.

Reply:
I have different taste here :) You need to decode it anyway, if you do it in the frontend, you couple the implementation of the Etag encoding of the version with the record editor logic.